# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/tests/unit/xpcshell.ini
# Commit: 16350e08d34f
# Full Hash: 16350e08d34ff06c7f46745c6b160c7a7bda9140
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2019-12-06 04:43:57
# Regressor Bug: 1586855
# File Overlap Count: 6
# Description:
#   bug 1586855 - incorporate CRLite filters into cert_storage r=jcj,kjacobs
#   
#   This patch implements CRLite lookups for TLS server certificate revocation
#   information in telemetry-only mode. It adds a new preference
#   "security.pki.crlite_mode" to control the behavior of this feature. Setting
# ==============================================================================

diff -r 64fd40663930 -r 16350e08d34f security/manager/ssl/tests/unit/xpcshell.ini
--- a/security/manager/ssl/tests/unit/xpcshell.ini	Thu Dec 05 22:23:54 2019 +0000
+++ b/security/manager/ssl/tests/unit/xpcshell.ini	Thu Dec 05 22:41:53 2019 +0000
@@ -14,7 +14,9 @@
   test_cert_overrides_read_only/**
   test_cert_sha1/**
   test_cert_signatures/**
+  test_cert_storage_direct/**
   test_cert_storage_preexisting/**
+  test_cert_storage_preexisting_crlite/**
   test_cert_trust/**
   test_cert_utf8/**
   test_cert_version/**
@@ -62,6 +64,7 @@
 [test_cert_storage_broken_db.js]
 [test_cert_storage_direct.js]
 [test_cert_storage_preexisting.js]
+[test_cert_storage_preexisting_crlite.js]
 # This test cannot succeed on 32-bit platforms. See bugs 1546361 and 1548956.
 skip-if = bits != 64
 [test_cert_storage_prefs.js]